app/components/admin/editions/show/preview_component.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class Admin::Editions::Show::PreviewComponent < ViewComponent::Base
+  include Admin::UrlOptionsHelper
+
+  attr_reader :edition
```

Comment:
Does this actually need an `attr_reader`? Is that just a style thing so we don't have to use `@` everywhere inside the class?

Reply:
Yea i should probably actually define it as private though so it's not available on the public interface. Will sort that now 👍 

Thanks for the review